sdks/java/io/tika/src/main/java/org/apache/beam/sdk/io/tika/TikaIO.java

```diff
@@ -63,82 +77,40 @@
    * A {@link PTransform} that parses one or more files and returns a bounded {@link PCollection}
    * containing one element for each sequence of characters reported by Apache Tika SAX Parser.
    */
-   public static Read read() {
-     return new AutoValue_TikaIO_Read.Builder()
-        .setQueuePollTime(Read.DEFAULT_QUEUE_POLL_TIME)
-        .setQueueMaxPollTime(Read.DEFAULT_QUEUE_MAX_POLL_TIME)
+   public static ParseAll parseAll() {
+     return new AutoValue_TikaIO_ParseAll.Builder()
         .build();
    }
 
    /** Implementation of {@link #read}. */
   @AutoValue
-  public abstract static class Read extends PTransform<PBegin, PCollection<String>> {
+  public abstract static class ParseAll extends
```

Comment:
Line 85: should reference parseAll instead

Reply:
Also, serialVersionUIDs throughout this PR are unnecessary and generally not used in most Beam code (with the exception of some low-level classes). These objects are not going to be serialized and then deserialized with a different version of the code.